mod/thewire/views/default/thewire.js

```diff
@@ -27,15 +27,16 @@ elgg.thewire.textCounter = function(textarea, status, limit) {
 
 	var remaining_chars = limit - $(textarea).val().length;
 	status.html(remaining_chars);
+	var $submit = $(textarea).closest('.elgg-form-thewire-add').find('#thewire-submit-button').eq(0);
```

Comment:
if you want to be sure with the span selector a few lines higher you would also be sure here right? I do not think it is needed here, but also not in the span situation...